english/macgyver/macgyver.md

```diff
@@ -14,3 +14,5 @@ Do you:
 [Jump off the cliff?](cliff/cliffjump.md)
 
 [Place the rope in wall like a wick and light it on fire?](light-rope-on-fire/light-rope-on-fire.md)
+
+[Tie the rope to the fork, stick it to the ceiling, climb up and burn a hole to escape?](ceiling_hole/ceiling_hole_burn.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.